.github/workflows/inference_unittests.yaml

```diff
@@ -0,0 +1,32 @@
+name: Unit Test for Inference
+
+on:
+  push:
+    branches:
+      - md-unittests
+    paths: 
+      - chatbot_inference/**
+      
+  pull_request:
+
+jobs:
+  build:
```

Comment:
unittest

Reply:
I have updated all the pipelines as per the standards. Please take a look at it.